dev/io.openliberty.jsonsupport.internal/resources/com/ibm/websphere/jsonsupport/resources/JSONSupportMessages_zh_TW.nlsprops

```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2015 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
```

Comment:
Also, if this is "new" then you'll want to remove the contributors part of the header.